api-reference/beta/api/directory_delete.md

```diff
@@ -0,0 +1,54 @@
+# Delete deletedItem
+
+Permanently delete an item in [deleted items](../resources/directory.md).
+
+### Prerequisites
+The following **scopes** are required to execute this API:
+* For users: *User.ReadWrite.All; Directory.Read.All; Directory.ReadWrite.All; Directory.AccessAsUser.All*
+* For groups: *Group.ReadWrite.All; Directory.Read.All; Directory.ReadWrite.All; Directory.AccessAsUser.All*
```

Comment:
Users is not supported yet.  Only groups AFAIK.  
And yes I believe it would be different depending on the entity being deleted.  I would stick with just showing the least privileged permission - like Group.ReadWrite.All.  User.ReadWrite.All is in development at the moment (but is not relevant because we only support this on Groups.